packages/web-components/src/components/va-table/va-table-inner/va-table-inner.scss

```diff
@@ -29,6 +29,18 @@
   }
 }
 
+// Using 29.99em instead of 480px because it matches USWDS' styles
+@media screen and (max-width: 29.99em) {
```

Comment:
Is it possible to use a token? (ie. `--mobile-lg`)

Reply:
@Andrew565 You mean these USWDS styles that are using `mobile-lg` before compiling? https://github.com/uswds/uswds/blob/a74f8aa92d803da949b642219623dee48dd05913/packages/uswds-core/src/styles/mixins/typography/usa-table-styles.scss#L18

The browser is showing the compiled value from the internal USWDS assignment so if you're just looking at the inspector, you might not be seeing the original assignment.

And if you don't mind mind, I think we should try to hold off on merging PRs until all of the conversations are resolved by the person asking the question (if there's not an urgent need for a merge).

Reply:
You're right, merged too soon, apologies. Yes, I was using just the browser inspector and only saw the compiled value. Using a token here would be the wiser thing to do than hardcoding a value. I'll spin up another PR to update this code.